frontend/src/lib/components/HtmlElementsDisplay/HtmlElementsDisplay.scss

```diff
@@ -0,0 +1,15 @@
+.HtmlElements {
```

Comment:
Nit:

```suggestion
.HtmlElementsDisplay {
```

Reply:
Took me ages to track down usage (or at least way longer than needed) -> renamed for clarity